### DISCUSSION TITLE
Combobox items not announced by screen readers

### DISCUSSION THREAD

**Discussion by abh1nash** [unovue/reka-ui#906] [Q&A]
I am using combobox as described in the documentation. I tried testing it with screen readers (NVDA and Talkback). I observed that while combobox itself is navigable and announced, the items that appear within the results are not announced. 

After digging a bit, I found that Radix Vue does not move focus to other list items upon navigation. I came to this conclusion after being unable to see a distinct black border that is created by Talkback on an item focused on the screen. While the hover state of the `ComboboxItem` changes, it does not receive focus required for the screen reader.

I tried to do the same with the example in the [official Radix Vue combobox docs](https://www.radix-vue.com/components/combobox.html) and found the same issue. However, things work well with [Headless UI's combobox component](https://headlessui.com/v1/vue/combobox).

Is this a bug?

**Comment by zernonia** [comment]
Hey @abh1nash ! Can you check again and if the issue still persist then it's a bug 🐛

**Reply by abh1nash** [reply]
Hi @zernonia Issue still there. For the combobox in the Radix Vue docs, it doesn't announce any of the list items.

**Reply by abh1nash** [reply]
I have created a pull request (#1002) that fixes this issue.
